pkgs/development/python-modules/pysrs/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, bsddb3
+}:
+
+buildPythonPackage rec {
+  pname = "pysrs";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-GmtjadY/rESlD3s2k4Ftm2xxYPCecLCyXs5XDJU1R9g=";
+  };
+
+  nativeBuildInputs = [ ];
+
+  propagatedBuildInputs = [ bsddb3 ];
+
+  preCheck = ''
+    rm -v testd.py testSRS.py
+  '';
```

```suggestion
  # Comment about why we can't run the tests
  doCheck = false;
```
Add `pythonImportsCheck`. E.g., something like:

```nix
pythonImportsCheck = [
    "SRS"
    "SES"
  ];
```